Automattic/jetpack
# Modules Admin: Allow VideoPress module to be managed for any plan

This is a follow-up to https://github.com/Automattic/jetpack/pull/21006 where we add support for the Jetpack VideoPress product and enable it for free sites. [We missed](https://github.com/Automattic/jetpack/pull/21006#pullrequestreview-764495803) that there was a separate module management page in Jetpack that needed changes to support this same functionality.

The VideoPress module will now be able to be activated by any plan, including free:

<img width="528" alt="Screen Shot 2021-09-27 at 11 28 05 AM" src="https://user-images.githubusercontent.com/789137/134956843-2248d0c0-d0b3-4c88-a51c-bb740826c608.png">

#### Changes proposed in this Pull Request:
* Enables the VideoPress module on the modules management view for any plan.

#### Jetpack product discussion
pxWta-Zz-p2

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Apply D66710-code to your wp.com sandbox if it hasn't been merged yet, sandbox `public-api.wordpress.com`, and enable `JETPACK__SANDBOX_DOMAIN` on your Jetpack dev site so that it points to your wp.com sandbox. You also may need to enable HTTP access on your WP.com sandbox.
* On your WordPress.com sandbox, enable the feature flag for Jetpack VideoPress in `mu-plugins/videopress/constants.php`: `define( 'VIDEOPRESS_JETPACK_VIDEO_ENABLED', true );`
* Remove all plan/upgrades so you are on the free Jetpack plan.
* Visit the dashboard, then click on `Modules` at the bottom of the screen.
* You should be able to activate the VideoPress module.
* Also try activating the module with the bulk-select tool, it should work.
* Head over to the media library and upload a video. If no videos were uploaded to the site previously, it should be uploaded successfully.
* Any video uploaded after the first upload should error out.
